src/main/java/educonnect/logic/parser/DeleteCommandParser.java

```diff
@@ -27,17 +28,17 @@ public DeleteCommand parse(String args) throws ParseException {
 
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_STUDENT_ID, PREFIX_EMAIL,
-                        PREFIX_TELEGRAM_HANDLE, PREFIX_TAG);
+                        PREFIX_TELEGRAM_HANDLE, PREFIX_TAG, PREFIX_TIMETABLE);
 
         // No prefixes used
         if (!areAnyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_TAG, PREFIX_STUDENT_ID, PREFIX_EMAIL,
-                PREFIX_TELEGRAM_HANDLE, PREFIX_TAG)) {
+                PREFIX_TELEGRAM_HANDLE, PREFIX_TAG, PREFIX_TIMETABLE)) {
```

Comment:
Hence it shouldn't be checked if its present.

Actually same applies to `Tag`, but others do confirm.

Reply:
This was on purpose so that the command parser can throw this error. Also I want to avoid situations where the user inputs `delete s/A1234567X c/[....] `

```
// Non-unique identifiers present
        if (areAnyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_TAG, PREFIX_TIMETABLE)) {
            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                    DeleteCommand.NO_UNIQUE_IDENTIFIER_MESSAGE));
        }
```

Reply:
ahh makes sense. that's actually a good approach and will adopt this for the feature I'm working on :)

otherwise, all LGTM